user_docs/en/changes.t2t

```diff
@@ -14,6 +14,7 @@ What's New in NVDA
 
 == Bug Fixes ==
 - Backspace now works correctly when using Nudi 6.1 with NVDA's "Handle keys from other applications" setting enabled. (#15822, @jcsteh)
+- Fixed a bug where audio coordinates would be played while the application is in sleep mode when "Play audio coordinates when mouse moves" is enabled. (#8059, hwf1324)

```

Comment:
@hwf1324 You forgot an @ (at sign) before your GitHub user name here.

Reply:
Thanks @lukaszgo1